html/browsers/browsing-the-web/history-traversal/resources/api-availability-1.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>API availability following history traversal - 1</title>
+<script>
+var controller = opener || parent;
```

Comment:
It cannot ever be parent, right? Let's remove that.

Reply:
Sure